app/tests/reader_studies_tests/test_forms.py

```diff
@@ -382,6 +396,48 @@ def test_image_port_only_with_bounding_box(
     assert Question.objects.all().count() == questions_created
 
 
+@pytest.mark.django_db
+@pytest.mark.parametrize(
+    "answer_type,ci_kinds", ANSWER_TYPE_TO_INTERFACE_KIND_MAP.items()
+)
+def test_question_interface_field(client, answer_type, ci_kinds):
```

Comment:
`client` isn't needed here. Can you do this without `@pytest.mark.django_db`? This test is going to be quite slow as it is creating a lot of `ci`s. I think there is also too much magic going on as it is constructing the test cases from the form of the object in the codebase. It's better if this is explicit.